cypress.json

```diff
@@ -1,7 +1,7 @@
 {
   "pluginsFile": "tests/e2e/plugins/index.js",
   "video": false,
-  "defaultCommandTimeout": 30000,
+  "defaultCommandTimeout": 10000,
```

Comment:
Our tests appear to be more resilient, so changing it back to 10 seconds to improve build time.